lib/development.conf

```diff
@@ -97,45 +97,52 @@ MODULES               = opencog/cogserver/server/libbuiltinreqs.so,
-                        scm/opencog/nlp/sureal.scm,
-                        scm/opencog/nlp/relex2logic.scm,
-                        scm/opencog/nlp/fuzzy.scm,
-                        scm/opencog/nlp/chatbot.scm
-
-#                        scm/config.scm,
-#                        server/scm/config.scm,
-#                        scm/core_types.scm,
-#                        spacetime/spacetime_types.scm,
-#                        nlp/types/nlp_types.scm,
-#                        dynamics/attention/attention_types.scm,
-#                        embodiment/embodiment_types.scm,
-#                        reasoning/pln/pln_types.scm,
-#                        scm/core-docs.scm,
-#                        scm/utilities.scm,
-#                        scm/apply.scm,
-#                        scm/file-utils.scm,
-#                        scm/persistence.scm,
-#                        scm/repl-shell.scm,
-#                        scm/av-tv.scm,
-#                        scm/rule-engine-utils.scm,
-#                        nlp/scm/type-definitions.scm,
-#                        nlp/scm/config.scm,
-#                        nlp/scm/file-utils.scm,
-#                        nlp/scm/nlp-utils.scm,
-#                        nlp/scm/disjunct-list.scm,
-#                        nlp/scm/processing-utils.scm,
-#                        nlp/relex2logic/utilities.scm,
-#                        nlp/relex2logic/rule-helpers.scm,
-#                        nlp/relex2logic/post-processing.scm,
-#                        nlp/lg-dict/utilities.scm,
+                        scm/core_types.scm,
+                        scm/core-docs.scm,
+                        scm/utilities.scm,
+                        scm/apply.scm,
+                        scm/file-utils.scm,
+                        scm/persistence.scm,
+                        scm/config.scm,
+                        scm/repl-shell.scm,
+                        scm/av-tv.scm,
+                        scm/rule-engine-utils.scm,
+                        dynamics/attention/attention_types.scm,
+                        embodiment/embodiment_types.scm,
+                        spacetime/spacetime_types.scm,
+                        nlp/types/nlp_types.scm,
+                        reasoning/pln/pln_types.scm,
+                        nlp/scm/type-definitions.scm,
+                        cogserver/server/scm/config.scm,
+                        nlp/scm/config.scm,
+                        nlp/scm/file-utils.scm,
+                        nlp/scm/nlp-utils.scm,
+                        nlp/scm/disjunct-list.scm,
+                        nlp/scm/processing-utils.scm,
+                        nlp/relex2logic/r2l-utilities.scm,
+                        nlp/relex2logic/rule-utils.scm,
+                        nlp/relex2logic/rule-helpers.scm,
+                        nlp/relex2logic/post-processing.scm,
+                        nlp/lg-dict/lg-dict.scm,
+                        nlp/sureal/surface-realization.scm
```

Comment:
In the long run, this should probably be replaced by a `(use-modules (opencog) (opencog cogserver) (opencog microplanning) ...etc.)`   I know that this doesn't quite always work as expected, due to various weirdnesses, but that should be the long-term goal.